src/osip-simulation-controller/src/main/resources/Bundle.properties

```diff
@@ -28,3 +28,19 @@ controller.helpdialog.controlTab.mixTankSection=Mix Tank Control
 controller.helpdialog.controlTab.mixTankText= The fourth tab is about the mix tank and thus doesn't have a slider for the input valve because the mix tank doesn't have an input valve and it doesn't have a temperature slider, as the input temperature is determined by the output temperature of the upper tanks. There is an output valve slider and additionally to the upper tanks, there is the possibility to select the motor speed of the mix motor.
 controller.helpdialog.settingsTab.header=Settings
 controller.helpdialog.settingslTab.generalText=The settings window can be opened via the "File" menu. It is necessary to set the ports for each tank, as every tank is implemented as its own OPC UA server. Therefore, there is an input field for each tank which you can use to set the port number.
+simulation.title=OSIP - Simulation
+label.tank.CYAN=Cyan Tank
+label.tank.YELLOW=Yellow Tank
+label.tank.MAGENTA=Magenta Tank
+label.tank.MIX=Mixtank
```

Comment:
Die Tank labels sollten ausschließlich in das Model. Sonst sind die doppelt, was unsere Erweiterbarkeit wieder kaputt macht. Hatten uns doch entschieden, dass nicht jeder selbst übersetzt, sondern dass wir die zentral haben...